tests/src/nesting_complex.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto2";
+
+package nesting_complex;
+
+// ----- Directly nested
```

Comment:
I would like to have more documentation explaining how this structure is nested.